scripts/python_submit_yarn.sh.template

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# This template is to show how to submit python job with dependencies on spark yarn clusters.
+
+
+PYSPARK_PYTHON=./ENV/venv/bin/python spark-submit \
+--conf spark.yarn.appMasterEnv.PYSPARK_PYTHON=./ENV/venv/bin/python \
```

Comment:
Seems like "appMasterEnv" would only set the driver env if we launch the application in yarn-cluster mode. Not sure if we can use```spark.executorEnv.[EnvironmentVariableName] 	``` in this case.